tools/jenkins/util.groovy

```diff
@@ -93,15 +105,16 @@ def integrationtest(display = 0) {
     }
 }
 
-def regression(build, env, display = 0) {
+def regression(build, env, modulepaths = [], display = 0) {
     cmd('Regression Tests', 'regress', ['DISPLAY=:' + display]) {
+        def modules = modulepaths ? '--modules ' + modulepaths.join(' ') : ''
         try {
             sh """
                 python3 ../inviwo/tools/regression.py \
                         --inviwo ../build/bin/inviwo \
                         --header ${env.JENKINS_HOME}/inviwo-config/header.html \
                         --output . \
-                        --repos ../inviwo
+                        --repos ../inviwo ${modules}
```

Comment:
A question that applies to some of the other tests as well:
Should inviwo directory always be included?
External modules only want to test their own repos, not inviwo (which should already be tested anyway)